src/sb_utils/launch/laptop_node.launch

```diff
@@ -0,0 +1,14 @@
+<!-- This launch file is for all the nodes/files running on the laptop:
+     pro_controller
+     realsense_both.launch
+     rviz to see where our rover is positioned
+     (We may need to run Gazebo to see how our arm is behaving). -->
+
+<launch>
+<!-- launch pro_controller -->
+<node pkg="procontroller_snowbots" type="pro_controller" name="pro_controller" output="screen"> </node>
+<!-- launch realsense_both.launch -->
+<include file="$(find cameras)/launch/realsense_both.launch"/>
+<!--launch rviz-->
+<node name="rviz" pkg="rviz" type="rviz"/>
```

Comment:
```suggestion
```
I know I mentioned we'll need `rviz` but for now you can remove it since the team is still determining how best to run rviz.